smart-contracts/scripts/multisig/create.js

```diff
@@ -1,8 +1,12 @@
 const { ethers } = require('hardhat')
 const { SafeFactory } = require('@gnosis.pm/safe-core-sdk')
 const EthersAdapter = require('@gnosis.pm/safe-ethers-lib').default
+const getOwners = require('./owners')
 
-async function main({ owners, threshold }) {
+const UNLOCK_MAINNET_MULTISIG_ADDRESS =
```

Comment:
Let's get it from the network package please? (and if it is not there, just add it there for all networks?)

Reply:
the other way is to get it with `unlock.owner()` which is maybe safer and prevents overloading  the networks package